aws_utils/s3/s3_utils.py

```diff
@@ -585,3 +586,30 @@ def upload_file(bucket, local_file_path, remote_destination_path):
     k = Key(bucket)
     k.key = remote_destination_path
     k.set_contents_from_filename(local_file_path)
+
+
+def get_latest_year_month_day_prefix(s3_path):
+    """Gets the date of the most recent year/month/day prefix in a S3 folder
+    Args:
+        s3_path (str): e.g s3n://audience-data-store-qa/artem/
+
+    Returns (Date): e.g datetime.date(2017, 5, 30)
+
+    """
+    latest = date.min()
+    keys = get_contents_of_directory(s3_path)
+
+    for key in keys:
+        search = re.search(r'.*year=(\d{4}).*month=(\d{2}).*day=(\d{2})', key)
+        if search:
+            year = int(search.group(1))
+            month = int(search.group(2))
+            day = int(search.group(3))
```

Comment:
there is the chance that this can be done this way:
`_, year, month, day = search.group`